lib/tasks/gibct.rake

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
```

Comment:
Do you think we still need the rake. I am not sure that this rake task would be terribly useful anymore, perhaps we should ignore it?